autoarray/plot/mat_plot/two_d.py

```diff
@@ -305,6 +311,11 @@ def plot_array(
         else:
             [text.set() for text in self.text]
 
+        if not isinstance(self.annotate, list):
+            self.annotate.set()
+        else:
+            [annotate.set() for annotate in self.annotate]

```

Comment:
Don't use comprehensions unless you're using the result. Make this a loop instead